### ISSUE TITLE
[20210801] Weekly AI ArXiv 만담

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#19]
- AI News
  - ACL 2021: 8.1 ~ 6, Tutorial, Main conference, Workshop
  - NeurIPS review: 우리시간 8월 4일 새벽이 될 듯 
  - CVPR2021 논문 흥행도: https://mattdeitke.com/cvpr-buzz/
  - OpenAI Triton: https://openai.com/blog/triton/
    - Document: https://triton-lang.org/
  - AI가 만든 지재권은? https://www.hankyung.com/politics/article/202107311971i 
  - 미중 AI 패권전쟁: https://www.hellot.net/news/article.html?no=60237
  - K-AI Hub: https://news.naver.com/main/read.naver?mode=LSD&mid=shm&sid1=105&oid=092&aid=0002229244
- ArXiv
  - [Go Wider Instead of Deeper](https://arxiv.org/abs/2107.11817)
    - MHSA 블럭내에 FC --> MoE 구조 
    - MHSA와 MoE는 block 간 sharing, LN만 블럭별로 --> 그렇게 파라미터를 절약
![image](https://user-images.githubusercontent.com/11782739/127733658-c0d68cef-3b90-467c-95fe-ab5668bed0b6.png)
![image](https://user-images.githubusercontent.com/11782739/127733713-1c0a6a50-0a13-4b92-9464-0c0a368fd3d4.png)

  - [FaVIQ: FAct Verification from Information-seeking Questions](https://arxiv.org/abs/2107.02153)
    - 새로운 Fact check dataset (고대 + UW)
    - https://faviq.github.io/   

  - [Pre-train, Prompt, and Predict: A Systematic Survey of Prompting Methods in Natural Language Processing](https://arxiv.org/abs/2107.13586v1)
    - Hyperscale LM 등장에 따른 새로운 패러다임인 prompt-based learning in PLM 정리
    - 관련 주제 연구자들이 집단지성으로 social research survey를 만드는 느낌.
    - 메인페이지에 연구 토픽별로 중요 논문들이 리스트업 --> Prompt-based Large-scale PLM 최신 논문이나 연구 파악에 유용해 보임
    - 본인의 연구논문을 paperlist에 직접 업로드 + promotion 가능
    - http://pretrain.nlpedia.ai/ 
![image](https://user-images.githubusercontent.com/11782739/127764959-003417dc-0184-4c14-a3b3-bbb9f7949977.png)
   

  - [Image Scene Graph Generation (SGG) Benchmark](https://arxiv.org/abs/2107.12604v1)
    - Image scene graph (object들의 attribute 와 relation 까지 detecting) generation 을 위한 벤치마크 + 코드 (from MS)
    - mask-rcnn + Visual Genome & Openimages data 
    - https://github.com/microsoft/scene_graph_benchmark
  
  - [Transfer Learning in Electronic Health Records through Clinical Concept Embedding](https://arxiv.org/abs/2107.12919v1)
    - EHR 데이터에 대한 pretraining embedding과 이에 기반한 transfer learning 분석 (U. of Oxford)
    - 질병 embedding 학습, embedding 평가 방법, 
    - 310만 환자 EHR 데이터 (CPRD 데이터)
![image](https://user-images.githubusercontent.com/11782739/127741301-0bb187b9-a432-46c9-b6e2-48cf1feba0ba.png)

  - [xMoCo: Cross Momentum Contrastive Learning for Open-Domain Question Answering](https://aclanthology.org/2021.acl-long.477.pdf)
    - Open domain QA의 dense retrieval model 학습을 위한 MoCo기반의 contrastive SSL
    - MoCo는 둘다 이미지지만 여긴 Question, Passage라 생겨 먹은게 달라서 모델 개선이 필요
    - 그래서 둘다 각각 fast / slow (moving average) encoder
![image](https://user-images.githubusercontent.com/11782739/127764241-d27d4a3e-4336-48a2-8ffc-aabaddaaa133.png)

  - [AutoTinyBERT: Automatic Hyper-parameter Optimization for Efficient Pre-trained Language Models](https://arxiv.org/abs/2107.13686v1)
    - 작은 스케일 PLM (BERT) 만들기 위한 evolutionary algorithm 활용 hyperparam optimization 기법 (화웨이+칭화대)
    - SuperPLM을 원샷으로 학습하고 latency constraint 걸어 모델을 깍아나가는 세팅
    - NAS-BERT나 DistillBERT보다 정확도도 좋고 효율적이라고..
    - https://github.com/huawei-noah/Pretrained-Language-Model/tree/master/AutoTinyBERT 
    - 실제 응용서비스 적용을 위해 PLM 깍으실 필요가 있는 분들께 유용할 듯?
![image](https://user-images.githubusercontent.com/11782739/127764706-dba07bc8-a3e9-48d4-b65a-66a5dd3f3d17.png)

**Comment by laray5959** [comment]
- [Audio-to-Score Alignment Using
Deep Automatic Music Transcription](https://arxiv.org/pdf/2107.12854.pdf)
![Audio-to-Score Alignment Using
Deep Automatic Music Transcription](https://github.com/laray5959/PaperList/blob/master/Audio-to-Score%20Alignment%20Using.PNG?raw=true)
    - Sample :
    - Conference : 
    - Problem :
    - Contribution 
        -

**Comment by veritas9872** [comment]
ICCV 2021 Accepted Papers List (Unofficial): https://github.com/amusi/ICCV2021-Papers-with-Code

ICCV 2021의 결과가 나오면서 많은 수의 Computer Vision Paper가 Arxiv에 업로드 되었습니다.

아직 공식적인 목록은 아니지만 비공식적으로 Accepted Papers List가 정리되었습니다.

많은 참조 부탁드립니다.

오늘은 논문 리뷰를 못해서 양해 부탁드립니다! :pray:

**Comment by jungwoo-ha** [comment]
@veritas9872 인간적으로 글로벌리 보는 거면 영어로좀 해주면 좋을텐데 말이죠 ㅎㅎㅎ

**Comment by Kyung-Min** [comment]
Paper
- CauseRec: Counterfactual User Sequence Synthesis for Sequential Recommendation (SIGIR 2021): https://dl.acm.org/doi/abs/10.1145/3404835.3462908
  - 추천시스템에서 유저 representation을 잘 학습하기 위해 observational 로그로부터 counterfactual 로그를 생성
  - counterfactual 로그는 observational 로그 중 일부 아이템에 intervention을 가한 것
  - observational 로그의 items를 크게 두 가지 타입으로 구분, indispensable/dispensable, 그리고 counterfactual 로그로써 negative/positive sample을 생성하기 위해 indispensable/dispensable 아이템을 각각 다른 아이템으로 replace 시킴
  - original/positive/negative sample로부터 metric learning 수행
  - ![image](https://user-images.githubusercontent.com/9531308/127771453-04b41f75-7481-4e52-86cd-0c98f67ddd34.png)

**Comment by Clyde21c** [comment]
[Med-BERT: pretrained contextualized embeddings on largescale structured electronic health records for disease prediction](https://www.nature.com/articles/s41746-021-00455-y)
![image](https://user-images.githubusercontent.com/35162035/127769788-a583858f-4d4a-440a-8493-761424cf1f90.png)
- npj digital medicine
- 미국내 600여개 병원의 2천만명 환자 EHR (Cerner)
- 진단 코드 이상의 context를 학습하기위한 Domain-specific pretraining task로서 입원기간 예측
- Fined-tunned task로는 당뇨병환자의  심부전 & 췌장암 예측
- 다른 task, 적은 데이터, 다른 EHR DB(Truven) 를 사용한 Pretrained EHR모델의 real-world generalizability 검증 의의

[Improving Early Sepsis Prediction with Multi Modal Learning](https://arxiv.org/abs/2107.11094)
- AWS AI Labs
- 폐혈증(sepsis)의 조기 예측
- multi modal 모델 : structured data에 의료진의 clinical note 추가
- 중환자실 오픈데이터 MIMIC-III

**Comment by hollobit** [comment]
Artificial intelligence can now be recognised as an inventor after historic Australian court decision
* https://www.abc.net.au/news/2021-08-01/historic-decision-allows-ai-to-be-recognised-as-an-inventor/100339264
*  "device for the autonomous bootstrapping of unified sentience", DABUS
* 호주 연방 법원의 조나단 비치 판사는 "발명가 ... 인공 지능 시스템 또는 장치가 될 수 있다"는 판결

Apple’s Machine Learning Researchers Have Developed A No-Code AI Platform Called ‘Trinity’ For Complex Spatial Datasets
* https://www.marktechpost.com/2021/07/23/apples-machine-learning-researchers-have-developed-a-no-code-ai-platform-called-trinity-for-complex-spatial-datasets/ 
* https://www.codingworldnews.com/news/articleView.html?idxno=4978 

NSF partnerships expand National AI Research Institutes to 40 states
* https://www.nsf.gov/news/news_summ.jsp?cntn_id=303176

Hundreds of AI tools have been built to catch covid. None of them helped 
* https://www.technologyreview.com/2021/07/30/1030329/machine-learning-ai-failed-covid-hospital-diagnosis-pandemic/ 
* https://www.turing.ac.uk/sites/default/files/2021-06/data-science-and-ai-in-the-age-of-covid_full-report_2.pdf
* https://www.bmj.com/content/369/bmj.m1328 

"AI와 데이터 분석은 개인화된 의료의 핵심 요소"...미국 의료계 대세가 된 AI
* http://www.aitimes.com/news/articleView.html?idxno=139910

**Comment by veritas9872** [comment]
제가 CVPR Buzz를 조금 읽어보고 있었는데 링크만 조금 바꾼다면 다른 학회 등에도 어렵지 않게 사용할 수 있을 것 같습니다. 잘하면 Arxiv Sanity 경쟁 웹사이트 만드실 수도 있을 것 같습니다. ㅋㅋㅋ
예를 들면 한국인 저자가 있는 논문만 리스트를 작성하거나 특정 대학교의 논문만 사용하는 방식으로 사이트를 새로 만들어볼 수 있을 것 같습니다.

**Comment by jungwoo-ha** [comment]
@veritas9872 네 소스코드도 공개되어 있어서 accepted paper list만 확보하면 어렵지 않게 가능할 것 같네용.

**Comment by veritas9872** [comment]
https://blogs.windows.com/windows-insider/2021/07/22/announcing-windows-11-insider-preview-build-22000-100/
Windows 11이 beta channel로 이동했습니다!
저도 지금 Windows 11에서 작업을 진행하는 중인데 딥러닝을 하는 연구자에게는 드디어 Windows에서 CUDA Docker를 사용할 수 있다는 점이라고 생각됩니다. 
실제로 PyTorch를 실행했을 때 잘 작동하는 것을 확인했습니다.
또한, WSL에서 Windows11에서 GUI 앱이 실행되니 윈도우 사용자 분들 사용해보시면 좋을 것 같습니다.
